src/Psalm/Internal/Stubs/CoreGenericFunctions.php

```diff
@@ -153,3 +153,13 @@ function array_chunk(array $arr, int $size, bool $preserve_keys = false) {}
  * @return bool
  */
 function array_key_exists($key, array $search) : bool { }
+
+/**
+ * @psalm-template TKey as array-key
+ * @psalm-template TValue
+ *
+ * @param array<TKey, TValue> $arr1
```

Comment:
```suggestion
 * @param array<TKey, TValue> $arr
```

Reply:
Nice one. Thanks.

Reply:
Yeah, this file is not checked by Psalm currently (because method bodies are missing) but probably should, at least in some guise